### ISSUE TITLE
Can not open sequence diagram

### ISSUE THREAD

**Issue by alepekhin** [argouml-tigris-org/argouml#59]
After building from source and lunching as java -jar argouml-with-dependencies.jar sequence diagram can not be created

**Comment by linustolke** [comment]
Yes, this is strange. Something has been broken. Let's hope that someone can understand what.
